apm-sniffer/apm-sdk-plugin/shenyu-2.4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/shenyu/v24x/ServerWebExchangeConstructorInterceptor.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.plugin.shenyu.v24x;
+
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
+
+/**
+ * ServerWebExchange Constructor Interceptor.
+ */
+public class ServerWebExchangeConstructorInterceptor implements InstanceConstructorInterceptor {
```

Comment:
This interceptor seems to do nothing, what is your plan for doing this?

Reply:
setSkyWalkingDynamicField(snapshot);
Later plugins can continue this snapshot when another thread executes.

Reply:
OK, then make sure the comments mentioned this. Otherwise, the codes look very strange.

Reply:
Very good suggestion, I have commented